.swiftlint.yml

```diff
@@ -1,70 +1,92 @@
-disabled_rules: # rule identifiers to exclude from running
-  - line_length
-  - trailing_whitespace
-  - todo # temporarily disabled. In general they should be treated as warnings
-#   - type_body_length # temporarily disabled.
-#   - file_length # temporarily disabled
-  - statement_position # wish we could support both uncuddled_else and default (but not others)
-  - trailing_comma
-  - syntactic_sugar
-  - redundant_optional_initialization
-  - redundant_void_return
-  - large_tuple # this is flagging closure argument declarations, seems like a mistake
-opt_in_rules: # some rules are only opt-in
-  - empty_count
-  # - missing_docs
-  # - force_unwrapping
-  # Find all the available rules by running:
-  # swiftlint rules
-# included: # paths to include during linting. `--path` is ignored if present.
-# #   - Source
-excluded: # paths to ignore during linting. Takes precedence over `included`.
-#   - Carthage
-  - LaunchDarkly/GeneratedCode
-  - Pods
-  - _iOS/Pods
-#   - Source/ExcludedFolder
-#   - Source/ExcludedFile.swift
+# See sub-configurations at `Source/.swiftlint.yml` and `Tests/.swiftlint.yml`.
 
-# configurable rules can be customized from this configuration file
-# binary rules can set their severity level
+disabled_rules:
+  - identifier_name
+  - weak_delegate
```

Comment:
Why not these too?

Reply:
`identifier_name` complains about any identifiers under 3 characters in length. So things like 
```
private let lf = Unicode.Scalar(0x0A)
```
get warnings. I personally feel this sort of case is justifiable, and we'll end up with `disable_rule`s as often as we want the rule to fire.

The second rule just seems a bit sketchy implementation wise to me. I think it's just matching on names that include `delegate`, which is likely to cause false positives.